pkg/azuredisk/azuredisk.go

```diff
@@ -358,3 +358,11 @@ func checkDiskName(diskName string) bool {
 
 	return true
 }
+
+func getResourceGroupFromDiskURI(diskURI string) (string, error) {
```

Comment:
pls also copy the unit test from cloud provider

Reply:
I have removed the function.